### PR TITLE
Greetings in any of your Favorite language

### PR DESCRIPTION
**Pull request by NatanimA**
### Project requirements

 - Create a new Rails app called 'hello-rails-react'.
      - Use the command `rails new -j webpack my_new_rails_app` as described in the previous lesson's tutorial.
 - Verify that the `jsbundling-rails` gem is added to your Gemfile.
 - Initialize your project with Git.
 - Install needed node dependencies as described in the previous lesson's tutorial.
 - Create a `babel.config.js` in the root folder of your project as described in the previous lesson's tutorial.
 - Update `webpack.config.js` to include the babel loader as described in the previous lesson's tutorial.
 - Make sure that your project has a Postgres database set up. 
 - Create a table for storing your messages and create 5 different greetings. 
 - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
 - Create a static view that will be the root of your app.
----
### TO RUN THE MIGRATIONS PLEASE FOLLOW THIS STEP

- [ ] **you need to enter your username and password in database.yml configuration in the test profile section**
- [ ] Then create the database ```rails db:create```
- [ ] Then Migrate ```rails db:migrate```
- [ ] Then Seed ```rails db:seeed```
- [ ] Then run ```rails s```
- [ ] Then Enjoy 😉 

blog_app_test

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -17,13 +17,13 @@ module.exports = {
...
   plugins: [
     new webpack.optimize.LimitChunkCountPlugin({
-      maxChunks: 1
-    })
-  ]
-}
+      maxChunks: 1,
+    }),
+  ],
+};
```
Comment (alexander16108) [inline webpack.config.js:29]: as required you are meant to have a babel.config.js config file setup using [this link](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @NatanimA , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ No Linter error
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 1 above)
Comment (NatanimA) [reply webpack.config.js:29]: How comes you cant see I have that Implemented? @alexander16108 
- Here is mine


- Here is the One on the link how comes you cant see it?

**Review by DammyShittu — APPROVED** [review]
Hi @NatanimA,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 1 above)
Comment (alexander16108) [reply webpack.config.js:29]: hi @NatanimA sorry for the mix up I do apologise for the error and thank you for pointing it out for me 
sorry and have a great week 
cheers 🥂

**Pull request merged**